lti_consumer/lti_consumer.py

```diff
@@ -558,6 +617,23 @@ def role(self):
         """
         return ROLE_MAP.get(self.runtime.get_user_role(), u'Student')
 
+    @property
+    def user_roles(self):
+        """
+        Get system user role and convert it to LTI 1.3 user roles.
```

Comment:
Seems the convert step is being performed somewhere else.

Reply:
Yeah, this can definitively be improved. I think I'll just pass the Django user role into the library.